app/assets/javascripts/routes/products_handlers.js

```diff
@@ -99,6 +100,11 @@ let ProductsHandlers = {
       type: ActionTypes.PRODUCT_RECEIVE,
       product: product
     });
+
+    Dispatcher.dispatch({
+      type: ActionTypes.BOUNTY_SELECTED_CHANGED,
+      id: bounty.id
```

Comment:
Expected ',' and instead saw 'id'.<br>Expected an identifier and instead saw ':'.<br>Expected ',' and instead saw 'bounty'.<br>Expected an identifier and instead saw '.'.<br>Too many errors. (43% scanned).